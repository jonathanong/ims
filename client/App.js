
import TopNav from 'ims-storybook/components/TopNav'
import { BrowserRouter } from 'react-router-dom'
import { hot } from 'react-hot-loader'
import React from 'react'

export const App = () => (
  <BrowserRouter>
    <TopNav />
  </BrowserRouter>
)

export default hot(module)(App)
