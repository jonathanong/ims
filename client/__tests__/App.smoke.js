
import React from 'react'
import { App } from '../App'
import renderer from 'react-test-renderer'

it('should render', () => {
  const tree = renderer
    .create(<App />)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
