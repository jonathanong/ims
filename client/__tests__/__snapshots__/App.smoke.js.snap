// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className={undefined}
>
  <a
    className={undefined}
    href="/"
    onClick={[Function]}
  >
    IMS
  </a>
</div>
`;
